wazo_calld/plugins/transfers/stasis.py

```diff
@@ -140,13 +148,17 @@ def process_answered_calls(self):
                 ):
                     channel = self.ari.channels.get(channelId=transfer.recipient_call)
                     if channel.json['state'] != 'Up':
-                        logger.debug('Recipiend answered from transfer %s', transfer.id)
+                        logger.debug('Recipient answered from transfer %s', transfer.id)
                         continue
                     event = {'args': ['', '', transfer.id]}
                     self.transfer_recipient_answered((channel, event))
-        logger.debug('Done.')
+                    answered_calls += 1
+        logger.debug(
+            'Finished processing %d answered calls since last stop.', answered_calls
+        )
 
     def stasis_start(self, event_objects, event):
+        logger.debug('received StasisStart event: %s', event)
```

Comment:
This is very noisy, since it also logs events unrelated to transfers. Please move the debug messages after verifying the event is related to transfers.